### PR TITLE
Fix objectlistview templates

### PR DESCRIPTION
**Pull request by yzaripova**
tfs #108016

### CODE REVIEW

File: addon/components/flexberry-menuitem.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ export default FlexberryBaseComponent.extend({
     // Store item object in DOM-element data attribute.
     // It will be used in root 'flexberry-menu' component to handle click on this 'flexberry-menuitem'.
     let item = this.get('item');
-    if (this.get('tagName')) {
+    if (this.get('tagName') !== '') {
```
Comment (hvostya) [inline addon/components/flexberry-menuitem.js]: Is it always not ''? Can it be undefined or null?

File: addon/components/flexberry-menuitem.js [hunk 2]
```diff
@@ -84,7 +84,7 @@ export default FlexberryBaseComponent.extend({
     this._super(...arguments);
 
     // Remove stored item object from DOM-element data attribute.
-    if (this.get('tagName')) {
+    if (this.get('tagName') !== '') {
```
Comment (hvostya) [inline addon/components/flexberry-menuitem.js]: Same question.

File: addon/components/flexberry-menuitem.js (hunk 1 above)
Comment (yzaripova) [inline addon/components/flexberry-menuitem.js]: Yes. In one case we have '' and can't continue.

File: addon/components/object-list-view.js [hunk 3]
```diff
@@ -551,14 +516,14 @@ export default FlexberryBaseComponent.extend(
...
-   *
-   * @property columnsCount
-   * @type Number
-   * @readonly
-   */
-  colspan: Ember.computed('columns.length', 'useSingleColumn', 'showHelperColumn', 'showMenuColumn', function() {
-    var columnsCount = 0;
+    Total columns count (including additional columns).
+
+    @property columnsCount
```
Comment (KollegOFF) [inline addon/components/object-list-view.js]: Typo.

Comment (yzaripova) [inline addon/components/object-list-view.js]: I don't see :(

Comment (KollegOFF) [inline addon/components/object-list-view.js]: Property name `colspan`, in docs `columnsCount`.

File: addon/components/mobile/object-list-view.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  @module ember-flexberry
+*/
+
+import Ember from 'ember';
+import ObjectListView from './../objectlistview';
+
+/**
+  Mobile version of objectlistview (with mobile-specific defaults).
+
+  @class Mobile.ObjectListView
```
Comment (hvostya) [inline addon/components/mobile/object-list-view.js]: It should be "ObjectListViewComponent".

File: addon/components/mobile/object-list-view.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
+/**
+  @module ember-flexberry
+*/
+
+import Ember from 'ember';
+import ObjectListView from './../objectlistview';
```
Comment (hvostya) [inline addon/components/mobile/object-list-view.js]: Canyou rewrite it like "import ObjectListViewComponent from '../objectlistview';"?

File: addon/components/mobile/object-list-view.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  @class Mobile.ObjectListView
+  @extends FlexberryObjectListView
+*/
+export default ObjectListView.extend({
+  /**
+    Flag indicates whether allow to resize columns (if `true`) or not (if `false`).
+
+    @property allowColumnResize
+    @type Boolean
+    @default true
```
Comment (hvostya) [inline addon/components/mobile/object-list-view.js]: false

File: addon/components/mobile/object-list-view.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+    @property singleColumnHeaderTitle
+    @type String
+  */
+  singleColumnHeaderTitle: undefined,
+
+  /**
+    Total columns count (including additional columns).
+
+    @property columnsCount
```
Comment (hvostya) [inline addon/components/mobile/object-list-view.js]: colspan?

File: addon/components/mobile/object-list-view.js (hunk 4 above)
Comment (yzaripova) [inline addon/components/mobile/object-list-view.js]: Yes, it is my mistake :)

File: addon/components/object-list-view.js (hunk 3 above)
Comment (yzaripova) [inline addon/components/object-list-view.js]: Thank!

File: addon/components/mobile/object-list-view.js (hunk 7 above)
Comment (yzaripova) [inline addon/components/mobile/object-list-view.js]: Thank!

File: addon/components/mobile/object-list-view.js (hunk 6 above)
Comment (yzaripova) [inline addon/components/mobile/object-list-view.js]: Thank!

File: addon/components/mobile/object-list-view.js [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+*/
+
+import Ember from 'ember';
+import ObjectListViewComponent from '../objectlistview';
+
+/**
+  Mobile version of {{#crossLink "ObjectListViewComponent"}}{{/crossLink}} (with mobile-specific defaults).
+
+  @class Mobile.ObjectListViewComponent
+  @extends FlexberryObjectListView
```
Comment (hvostya) [inline addon/components/mobile/object-list-view.js]: Really? Not ObjectListViewComponent?

File: addon/components/object-list-view.js [hunk 9]
```diff
@@ -22,6 +22,16 @@ export default FlexberryBaseComponent.extend(
   FlexberryFileCompatibleComponentMixin,
   ErrorableControllerMixin, {
   /**
+    Projection set by property `modelProjection`.
```
Comment (hvostya) [inline addon/components/object-list-view.js]: You can do reference on property here.

File: addon/components/object-list-view.js [hunk 10]
```diff
@@ -166,12 +201,12 @@ export default FlexberryBaseComponent.extend(
   }),
 
   /**
-    Path to component's settings in application configuration (JSON from ./config/environment.js).
-
-    @property appConfigSettingsPath
-    @type String
-    @default 'APP.components.flexberryBaseComponent'
-  */
+   * Path to component's settings in application configuration (JSON from ./config/environment.js).
```
Comment (hvostya) [inline addon/components/object-list-view.js]: Why are stars used on comment?

**Pull request merged**